tests/unit/test_cocip_grid.py

```diff
@@ -773,3 +741,14 @@ def test_verbose_outputs_formation(
     assert ds["fuel_flow"].mean() == pytest.approx(0.6037, rel=rel)
     assert ds["rhi"].mean() == pytest.approx(0.6273, rel=rel)
     assert ds["iwc"].mean() == pytest.approx(4.4621e-06, rel=rel)
+
+
+def test_max_age_exceeds_met(instance_params: dict[str, Any], source: MetDataset) -> None:
+    """Perform smoke test to ensure max_age can exceed the available met time."""
```

Comment:
For my understanding, what actually happens in this case? Is the met data extended somehow, or do you fall back to the last available time?

Reply:
The model proceeds as normal from the start. Eventually we end up with a `GeoVectorDataset` whose time column exceeds the time dimension in `met`. When everything funnels down into the `RegularGridInterpolator`, the points will be out of bounds for the grid. With default parameters, we'll get nans. If `params["interpolation_bounds_error"] == True`, we'll get error. And if `params["interpolation_fill_value"] == None`, the value is extrapolated (we haven't played with this setting much).

In the default case, when we get a nan at the waypoint from interpolation, this nan will propagate forward into all the CoCiP calculations. Eventually there is a survival check, and the nan values will trigger an end of life condition. So the contrail waypoint will stop evolving.